### PR TITLE
Add jax.numpy.atleast_2d Frontend

### PR DESCRIPTION
**Pull request by mohamedehab0999**

### CODE REVIEW

**Comment by mohamedehab0999** [comment]
Closes https://github.com/unifyai/ivy/issues/9355
Tracking TODO issue https://github.com/unifyai/ivy/issues/8422

**Comment by ivy-seed** [comment]
This PR has been labelled as stale because it has been inactive for more than 7 days. If you would like to continue working on this PR, then please add another comment or this PR will be closed in 7 days.

**Comment by vaatsalya123** [comment]
Hi @mohamedehab0999 the PR looks great though just a small change is required since there is a failing test due to it, Just change your testing code to use test_flags instead once you make that change we can merge this PR thanks!

**Comment by mohamedehab0999** [comment]
@vaatsalya123 Done

**Comment by vaatsalya123** [comment]
lgtm!
